paddle/operators/cross_entropy_op.h

```diff
@@ -21,25 +21,53 @@ namespace operators {
 template <typename Place, typename T>
 class OnehotCrossEntropyOpKernel : public OpKernel {
 public:
-  constexpr T LOG_THRESHOLD() const { return static_cast<T>(1e-20); }
+  constexpr T kLOG_THRESHOLD() const { return static_cast<T>(1e-20); }
```

Comment:
Let's follow the code style https://google.github.io/styleguide/cppguide.html#Constant_Names and change `kLOG_THRESHOLD` into 

```cpp
static const double kLogThreshold = 1e-20;
```

We mustn't define it as a function template without any constaint on type `T`, as it is obvious that we cannoot type cast 1e-20 into any integral value like `LOG_THRESHOLD<int>()`.

Let's fix this special case in this PR and fix others in another separate PR.